### ISSUE TITLE
Change color of dot

### ISSUE THREAD

**Issue by criss02-cs** [yurkinh/Plugin.Maui.Calendar#7]
I would like to change the color of the point of a single event, in the examples only groups of events are changed

**Comment by yurkinh** [comment]
Hi @criss02-cs 
Have you checked AdvancedPage sample?
Can you provide an example?

**Comment by criss02-cs** [comment]
Hi, I have checked advancedPage sample.

```
Events = new EventCollection
        {
            [DateTime.Now.AddDays(-3)] = new List<AdvancedEventModel>(GenerateEvents(10, "Cool")),
            [DateTime.Now.AddDays(-6)] = new DayEventCollection<AdvancedEventModel>(Colors.Purple, Colors.Purple)
            {
                new AdvancedEventModel
                {
                    Name = "Cool event1", Description = "This is Cool event1's description!", Starting = new DateTime(), Finishing = new DateTime()
                },
                new AdvancedEventModel
                {
                    Name = "Cool event2", Description = "This is Cool event2's description!", Starting = new DateTime(), Finishing = new DateTime()
                }
            }
        };
```
In that sample with DayEventCollection it is possible to change the color, but it concerns only the day. What I want to do is to change color for every event

**Comment by yurkinh** [comment]
![Simulator Screenshot - iPhone 15 - 2023-12-20 at 10 10 14](https://github.com/yurkinh/Plugin.Maui.Calendar/assets/17849938/62cbd885-232e-4149-9eaa-e181fd60bd10)


Here is a screenshot from SimplePage where Today's event indicator has different color from other days indicator

**Comment by criss02-cs** [comment]
That's fine. But what if I have 2 events for today? I want to have different dots color

**Comment by yurkinh** [comment]
Do you mean two different dot indicators (dots) for one day?

**Comment by criss02-cs** [comment]
Yes

**Comment by yurkinh** [comment]
I will take a look at what we can do to solve this

**Comment by yurkinh** [comment]
@criss02-cs  I think that we can put max 3 dots with different colors. Will it be enough for you? 
Otherwise it will look ugly

**Comment by criss02-cs** [comment]
Absolutely

**Comment by criss02-cs** [comment]
Do you have any updates on this?

**Comment by yurkinh** [comment]
In progress. I need first to finish other issues but I already have few ideas how to correctly implement this

**Comment by criss02-cs** [comment]
Just so you know, do you think you'll be able to finish it by mid-January? I would have a deadline and I would like there to be this feature, but it's not urgent

**Comment by yurkinh** [comment]
Hi @criss02-cs 
Hope so to finish it by this time. I first want to close critical issues with performance and bugs and then continue feature implementation

**Comment by criss02-cs** [comment]
Hi @yurkinh 
No problem

**Comment by mansley93854** [comment]
Any updates on the multiple dot issue?

**Comment by yurkinh** [comment]
Hi @mansley93854 
This functionality will be available in the next major release v2.0.0, which is planed for the nearest future

**Comment by yurkinh** [comment]
Hi @criss02-cs @mansley93854 
Please check the latest version v1.2.3.
it has support for the multiple events per day
